model/src/test/java/com/javahelp/model/usertest/UserEntityTest.java

```diff
@@ -1,25 +1,25 @@
-package com.javahelp.model.userEntityTest;
+package com.javahelp.model.usertest;
 
+import com.javahelp.model.user.User;
 import com.javahelp.model.user.UserInfo;
 import com.javahelp.model.user.ClientUserInfo;
-import com.javahelp.model.user.UserType;
-import com.javahelp.model.user.ProviderUserInfo;
 
 import org.junit.*;
 
 import static org.junit.Assert.*;
 
+/**
+ * These are tests for User Entity
+ */
 public class UserEntityTest {
 
-    /** These are tests for User Entity
-     *
+    /**
      * --- Attributes ----
      * user1 - The first user object
      * user2 - the second user object
      * userinfo1 - the information of the first user
      * userinfo2 - the information of the second user
      */
```

Comment:
This comment is kinda unconventional for Java. Instead of one big comment for all fields, the convention is to write a single Javadoc for each public facing field. In this case, it's not really necessary.